app/charts/shared/chart-data-filters.tsx

```diff
@@ -145,6 +149,18 @@ const DataFilter = ({
     },
   });
 
+  const hierarchyParents = useHierarchyParents(
+    dataSetIri,
+    dataSource,
+    data?.dataCubeByIri?.dimensionByIri!,
+    locale,
+    !data?.dataCubeByIri?.dimensionByIri
+  );
+
```

Comment:
Yes the object parameter would be better. But dimensionByIri should not be passed "once", here the second times is for the pause argument. In another usage, we use the pause argument to pause the request until the select is opened. It will be clearer with the object options.